content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Enabling Multicluster on IBM Cloud Private
+description: Example multicluster IBM Cloud Private install of Istio.
+weight: 70
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use Istio' multicluster feature to join two
+[IBM Cloud Private](https://www.ibm.com/cloud/private) clusters together,
+using the [kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Create the IBM Cloud Private Clusters
+
+1.  You can follow steps [here](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_2.1.0.3/installing/installing.html)
+    to install two IBM Cloud Private clusters.
+    __NOTE__: Make sure individual cluster Pod CIDR ranges and service CIDR ranges must be unique across the multicluster
+    environment and may not overlap. This can be configured by `network_cidr` and `service_cluster_ip_range` in `cluster/config.yaml`.
+    
+    {{< text bash >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  After IBM Cloud Private cluster install finsihed, validate `kubectl` access to each cluster. Suppose we have
+    two clusters: cluster-1 and cluster-2.
```

Comment:
Line break:

In this example, consider two clusters `cluster-1` and `cluster-2`:

Reply:
Done